### ISSUE TITLE
[Template] Wild Question

### ISSUE THREAD

**Issue by mguidoti** [gsautter/goldengate-imagine#870]
Hi Guido,

Just one wild question about how GGI works: if I've a template loaded in a file and run the steps manually, will this simulate the batch processing, meaning that, the changes I make in the template can affect the outcome of the 'Document Structure' tool?

Thanks

**Comment by gsautter** [comment]
Oh yes it will ... the gizmo is the very same, and so is its template usage ... the only difference is the presence or absence of the UI.

**Comment by mguidoti** [comment]
OK, good, this is what I thought. 

Ok. So before batch processing we will run a couple of steps on sample PDFs for the different journals/templates we've been working on. We might report here what we're finding, so we can understand if its major or minor issues, and if we can carry on with the batch process. Maybe @myrmoteras can weight-in.

Would that be all rightw

**Comment by gsautter** [comment]
Yeah, running the individual steps in the UI definitely helps. Also, you can "Undo", modify (and save) the template, and re-run the step, repeating until you're satisfied with the result ... which is also what I used to do when making the first few templates.

"Tools > Check Document Errors" might help identify issues, as it does the exact same as the QC (it _is_ the QC routine, for that matter).

In case you find anything, a first test might be running it _without_ a template to exclude the latter as the source of the error. This will take a second GGI installation or starting with a different configuration, as once a template is known, all the gizmos use it.

**Comment by mguidoti** [comment]
Even if we exclude the association between the template and the given article? Wouldn't this avoid the need for a secondary installation?

**Comment by gsautter** [comment]
To exclude association, you'd have remove the respective document attributes (`docStyle` and `docStyleName`) via "Edit > Edit Document Attributes" (the easy part), and you'd have to distort any anchors to prevent automated re-association of the template (the impractical, disturbing part).

In the light of the latter, I was suggesting a dedicated "test without templates" installation ... but another _configuration_ should also do, as the templates are stored inside the configuration you have open when creating them (to change the latter is a completely separate issue).
